pkg/audit/manager.go

```diff
@@ -714,6 +714,8 @@ func logConstraint(l logr.Logger, constraint *unstructured.Unstructured, enforce
 	l.Info(
 		"audit results for constraint",
 		logging.EventType, "constraint_audited",
+		logging.ConstraintGroup, constraint.GroupVersionKind().Group,
```

Comment:
In addition to adding this for audit logs, we should also add it for
constraint controller:
https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/controller/constraint/constraint_controller.go#L353-L370
and admission
https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/webhook/policy.go#L240-L263

Reply:
Done